packages/config/src/chains.ts

```diff
@@ -662,16 +670,32 @@ export const moonbaseAlpha = new EvmParachain({
 });
 
 export const moonbaseBeta = new EvmParachain({
+  assetsData: [
+    {
+      asset: betaDEV,
+      metadataId: 0, // no metadata for native tokens
+    },
+    {
+      asset: dev,
+      balanceId: '222902676330054289648817870329963141953',
+      id: { ForeignAsset: '222902676330054289648817870329963141953' },
+    },
+    {
+      asset: alan,
+      balanceId: '85534404031760856987006367174489651085',
+      id: { ForeignAsset: '85534404031760856987006367174489651085' },
+    },
+  ],
   ecosystem: Ecosystem.AlphanetRelay,
   genesisHash:
     '0xeebb5d05763801e54d6a7a60a4b7998ac125c4d050dcec418dd07ea959a54464',
-  id: 1287,
+  id: 1282,
   isTestChain: true,
   key: 'moonbase-beta',
   name: 'Moonbase Beta',
   parachainId: 888,
   rpc: 'https://rpc.api.moondev.network',
-  ss58Format: 1287,
+  ss58Format: 1282,
```

Comment:
is this correct? 
![image](https://github.com/PureStake/xcm-sdk/assets/93129175/8f40b48a-b87c-48cd-a678-ffe0c26a916f)

Reply:
The same.